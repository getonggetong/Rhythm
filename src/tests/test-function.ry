
getNotes() {
	return [A4,B4,C4];
}

plusOne(e1) {
	return e1 + 1;
}

add(e1,e2) {
	return e1 + e2;
}



init() {
	a = getNotes();
	b = plusOne(a);
	c = add(a,b);
	print(a);
	print(b);
	print(c);
}