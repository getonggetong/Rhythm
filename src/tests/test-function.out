[A4,B4,C4]
[A#4.4,C4.4,C#4.4]
[[A4,A#4.4],[B4,C4.4],[C4,C#4.4]]